subworkflows/local/domain_annotation/meta.yml

```diff
@@ -0,0 +1,60 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
+name: "domain_annotation"
+description: Annotate amino acid fasta files with selected HMM libraries such as Pfam and FunFam
+keywords:
+  - fasta
+  - sequences
+  - domain
+  - annotation
+  - database
+  - download
+  - HMM
+components:
+  - aria2
+  - hmmer/hmmsearch
+input:
+  - ch_fasta:
+      type: file
+      description: |
+        Amino acid fasta file containing amino acid sequences for annotation
+  - skip_pfam:
+      type: boolean
+      description: |
+        Skip domain annotation with Pfam
+  - pfam_db:
+      type: string
+      description: |
+        Path to an existing HMM Pfam library on the system
```

Comment:
Could you add something like: "if provided, the `ARIA2_PFAM` db download will be skipped" ?